sfge/src/test/java/com/salesforce/rules/unusedmethod/UnusedMethodRuleTest_EdgeCases.java

```diff
@@ -0,0 +1,199 @@
+package com.salesforce.rules.unusedmethod;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+/** A collection of tests for weird edge cases. */
+public class UnusedMethodRuleTest_EdgeCases extends UnusedMethodRuleTest_BaseClass {
```

Comment:
Can we also follow the standard for other classes please? Would be nice to have all the names end with "Test".
Since the package already mentioned "unusedmethod", do we need to reiterate that in the class' name? I've been meaning to fix this for FLS tests as well.

Reply:
Same for the other test classes in this PR.